README.md

```diff
@@ -30,13 +30,11 @@ identity providers are under active development.
 
 ## Documentation
 
-Documentation for authd is currently available as a [wiki](https://github.com/ubuntu/authd/wiki/01---Get-started-with-authd) that includes how-to guides on:
+If you want to know more about using authd, refer to the
+[official authd documentation](https://canonical-authd.readthedocs-hosted.com/en/latest/).
```

Comment:
As per the previous commit suggestion, you can then avoid repeating the same but use the reference link instead.

Reply:
Can do this too? :)

Reply:
My apologies! Done in https://github.com/ubuntu/authd/pull/671/commits/7dd469282b654b5962ac8c2cb03c344beef880ae